retail/clickstream/cloud-run/datalayer-pubsub/.dockerignore

```diff
@@ -0,0 +1,4 @@
+Dockerfile
+.dockerignore
+node_modules
+npm-debug.log
```

Comment:
Please add new line at the end of the files.

Reply:
Done!